salt/states/lxc.py

```diff
@@ -681,8 +681,10 @@ def edited_conf(name, lxc_conf=None, lxc_conf_unset=None):
         This state is unsuitable for setting parameters that appear more than
         once in an LXC config file, or parameters which must appear in a
         certain order (such as when configuring more than one network
-        interface). It is slated to be replaced, and as of version 2015.5.0 it
-        is deprecated.
+        interface).
+
+        Issue #35523 was opened to track the addition of a suitable replacement
```

Comment:
You can link to this issue by making an RST link like so:

```
`Issue #35523`_ was opened to track the addition of a suitable replacement
or fix

.. _`Issue #35523`: https://github.com/saltstack/salt/issues/35523
```

Reply:
Done.